commands/music.py

```diff
@@ -17,7 +17,9 @@
 # Please ensure that you have enabled the YouTube Data API for your project.
 
 # Make sure you set your environment variable
-DEVELOPER_KEY = os.environ['YT_API']
+config = configparser.ConfigParser()
+config.read('config/config.ini')
```

Comment:
Don't read multiple times, make an import like `from bot import config` and use that wherever the config is required.

Reply:
Done - code can be found in configstartup.py